### ISSUE TITLE
Cannot remove Notification Item

### ISSUE THREAD

**Issue by perfectstrong** [os-js/OS.js#640]
## Expected behaviour
I'm trying to remove the inserted icon by `Notification.destroyIcon(NotiName)`

## Actual behaviour

The return was `true`, except the icon didnot move.

## Steps to reproduce the error
```
const Notification = OSjs.require('gui/notification');
const NotiName = '_ShieldNotification';
.
.
Notification.destroyIcon(NotiName); // Returns true but does nothing
```

**Comment by perfectstrong** [comment]
By digging into `gui/notification.js`, i figured that the method `destroyIcon()` misses the arguement `name`.

**Comment by andersevenrud** [comment]
Ops! `name` is actually a global variable for some reason, hence why I've never caught this.

Feel free to fix it and make a PR to `development` ;)

**Comment by andersevenrud** [comment]
This was fixed in #646

**Comment by andersevenrud** [comment]
Now in master :)
